exs/mundo_1/ruby/020.rb

```diff
@@ -0,0 +1,19 @@
+=begin
+Desafio 020
+Problema: O mesmo professor do desafio 019 quer sortear a ordem
+          de apresentação de trabalhos dos alunos. Faça um programa
+          que leia o nome dos quatro alunos e mostre a ordem sorteada.
+Resolução do problema:
+=end
```

Comment:
E o último de espaçamento para corrigir

```suggestion
=begin
Desafio 020

Problema: O mesmo professor do desafio 019 quer sortear a ordem
          de apresentação de trabalhos dos alunos. Faça um programa
          que leia o nome dos quatro alunos e mostre a ordem sorteada.
          
Resolução do problema:
=end
```